data/json/test3.jsn

```diff
@@ -0,0 +1 @@
+{"chilllllllllllllllllllllllld":"Niñññññññññññññññññññññññño","child-jp":"子供","child":"Niño"}
```

Comment:
newline or nah?

Reply:
in this case i'd say no since we're merging upstream and I don't want to cause future merge conflicts